File: 2021/1-1. Sonar Sweep/src/main.cpp
#include <fstream>
#include <iostream>

int main()
{
    std::ifstream infile("input.txt");

    if (!infile.is_open())
    {
        std::cerr << "ERROR! Could not open input.txt!\n";
        return -1;
    }

    int prev = 0;
    int current = 0;

    int increases = 0;

    infile >> prev;

    while (infile >> current)
    {
        if (current > prev)
            increases++;

        prev = current;
    }

    std::cout << "Answer: " << increases << '\n';

    return 0;
}

File: 2021/1-1. Sonar Sweep/Python/main.py
infile = open("input.txt")

prev: int = int(infile.readline())
current: int = 0

increases: int = 0

while True:
    try:
        current = int(infile.readline())
    except:
        break

    if current > prev:
        increases += 1

    prev = current

print("Answer:", increases)

infile.close()